app/tests/spec/lib/app-start.js

```diff
@@ -132,6 +132,19 @@ function (chai, sinon, AppStart, Session, Constants, p,
                       assert.ok(userMock.upgradeFromSession.calledOnce);
                     });
       });
+
+      it('tracks window errors', function () {
+        return appStart.startApp()
+          .then(function () {
+            sinon.stub(appStart._metrics, 'logEvent', function () {
+            });
+
+            window.onerror.call(window, 'message', document.location.toString(), 2);
+          })
+          .then(function () {
+            assert.ok(appStart._metrics.logEvent.calledOnce);
```

Comment:
We have a [helper](https://github.com/mozilla/fxa-content-server/blob/382c9db980753300b38b3f767aa071a1163ccf73/app/tests/lib/helpers.js#L91) that allows you to test for specific events in the event log. For an example of use, see [this snippet](https://github.com/mozilla/fxa-content-server/blob/382c9db980753300b38b3f767aa071a1163ccf73/app/tests/spec/views/sign_in.js#L204-L205).